docs/modelserving/observability/grafana_dashboards.md

```diff
@@ -0,0 +1,39 @@
+# Grafana Dashboards 
+
+Some example Grafana dashboards are available in GrafanaLabs. 
+
+## Knative HTTP Dashboard (if using serverless mode)
+
+The [Knative HTTP Grafana dasbhoard](https://grafana.com/grafana/dashboards/18032-knative-serving-revision-http-requests/) was built from [Knative's sandbox monitoring example]https://github.com/knative-sandbox/monitoring().
+
+## KServe ModelServer Latency Dashboard
+
+A template dashboard for [KServe ModelServer Latency](https://grafana.com/grafana/dashboards/17969-kserve-modelserver-latency/) contains example queries using the prometheus metrics for pre/post-process, predict and explain in milliseconds. The query is a [histogram quantile](https://prometheus.io/docs/prometheus/latest/querying/functions/#histogram_quantile). A fifth graph shows the total number of requests to the predict endpoint. This graph covers all KServe's ModelServer runtimes - lgbserver, paddleserver, pmmlserver, sklearnserver, xgbserver, custom transformer/predictor.
+
+## KServe TorchServe Latency Dashboard
+
+A template dashboard for [KServe TorchServe Latency](https://grafana.com/grafana/dashboards/18026-kserve-torchserve-latency/) contains an inference latency graph which plots the [rate](https://prometheus.io/docs/prometheus/latest/querying/functions/#rate) of the TorchServe metric `ts_inference_latency_microseconds` in milliseconds. A second graph plots the rate of TorchServe's internal queue latency metric `ts_queue_latency_microseconds` in milliseconds. A third graph plots the total requests to the TorchServe Inference Service. 
```

Comment:
Let's link to the TorchServe metrics reference https://pytorch.org/serve/metrics.html, seems like there are more metrics like memory/CPU/GPU we should add to the dashboard.

Reply:
ok interesting because when I hit the endpoint none of those are exposed

Reply:
I see these though https://pytorch.org/serve/metrics_api.html